examples/contracts/cw1-whitelist/src/contract.rs

```diff
@@ -12,30 +12,41 @@ use sylvia::entry_points;
 const CONTRACT_NAME: &str = env!("CARGO_PKG_NAME");
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
-pub struct Cw1WhitelistContract {
+#[cosmwasm_schema::cw_serde]
+pub struct SvCustomQuery;
+impl cosmwasm_std::CustomQuery for SvCustomQuery {}
+
+pub struct Cw1WhitelistContract<E, Q> {
     pub(crate) admins: Map<&'static Addr, Empty>,
     pub(crate) mutable: Item<bool>,
+    pub(crate) _phantom: std::marker::PhantomData<(E, Q)>,
 }
 
-#[cfg_attr(not(feature = "library"), entry_points)]
+#[cfg_attr(not(feature = "library"), entry_points(generics<Empty, SvCustomQuery>))]
```

Comment:
We want to make the contract generic for other users to test this contract with different CustomMsg and CustomQuery.
Same for other CW contracts.

```suggestion
#[cfg_attr(not(feature = "library"), entry_points(generics<Empty, Empty>))]
```